files/checks/check_systemd_service.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+[ -n "$2" ] || exit 1;
+
+is_active=$(systemctl is-active "$2");
+echo "$2 is $is_active on $(hostname)"
```

Comment:
According to the [docs](https://nagios-plugins.org/doc/guidelines.html#AEN33), the output should be of the form `SERVICE STATUS: Information text`.